app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -77,12 +74,6 @@ class MainActivity : AppCompatActivity() {
                 navigation.selectedItemId = navigation_search
             }
 
-            if (bundle.getBoolean(LAUNCH_ATTENDEE)) {
-                val fragment = AttendeeFragment()
-                fragment.arguments = bundle
-                loadFragment(supportFragmentManager, fragment, rootLayout.id)
-            }
-
```

Comment:
How is this case handled?

Reply:
> How is this case handled?

It is not required now. Because here we start auth activity from the fragment, authenticate user, finish the auth activity and continue with the attendee fragment. No need to open main activity, pass bundle and load fragment again with same data.